src/EnergyPlus/DataHVACGlobals.cc

```diff
@@ -100,12 +100,10 @@ namespace DataHVACGlobals {
     int const CoolingWaterDesAirOutletTempSizing(7);                 // request sizing for cooling water coil outlet air temp
     int const CoolingWaterDesAirOutletHumRatSizing(8);               // request sizing for cooling water coil outlet air humidity ratio
     int const CoolingWaterNumofTubesPerRowSizing(9);                 // request sizing for cooling water coil number of tubes per row
-    int const HeatingWaterDesAirInletTempSizing(10);                 // request sizing for heating water coil inlet air temp
```

Comment:
Renumber these?

Reply:
No, I don't think that's necessary.  They'll all be gone when we're done, so the number won't matter.